[Nuclio] Invocation URL: N/A on error or not exposed, +info

skavronsky opened the pull request.

mavdryk commented on src/nuclio/common/services/version-helper.service.js:
```diff
@@ -21,6 +21,7 @@
          * @param {Object} version
          * @returns {{text: string, valid: boolean}} URL invocation data
          */
+        /*eslint complexity: ["error", 12]*/
```
move to the beginning of the file

mavdryk commented on src/nuclio/common/services/version-helper.service.js:
```diff
@@ -31,25 +32,28 @@
                     .get('[0][1].host')              // host
                     .value();
 
+                var serviceType = lodash.get(httpTrigger, 'attributes.serviceType');
```
there is no need to move this line over here

mavdryk commented on src/nuclio/common/services/version-helper.service.js:
```diff
@@ -31,25 +32,28 @@
                     (serviceType === 'NodePort' || !FunctionsService.isKubePlatform()) &&
                     httpPort !== 0 &&
-                    !lodash.isEmpty(externalIpAddress)
+                    !lodash.isEmpty(externalIpAddress) && serviceType !== 'ClusterIP' &&
```
move `serviceType !== 'ClusterIP'` to the new line

mavdryk commented on src/nuclio/common/services/version-helper.service.js:
```diff
@@ -59,7 +63,7 @@
 
             return {
-                text: $i18next.t('functions:URL_NOT_EXPOSED', { lng: i18next.language }),
+                text: $i18next.t('functions:common:N_A', { lng: i18next.language }),
```
there is a mistake. 2 namespaces are used

mavdryk commented on src/nuclio/functions/function-collapsing-row/function-collapsing-row.component.js:
```diff
@@ -19,6 +19,7 @@
             controller: NclFunctionCollapsingRowController
         });
 
+    /*eslint max-statements: ["error", 51]*/
```
move to the beginning of the file

mavdryk commented on src/nuclio/functions/function-collapsing-row/function-collapsing-row.component.js:
```diff
@@ -192,6 +194,10 @@
          * @param {string} state - absolute state name or relative state path
          */
         function onSelectRow(event, state) {
+            if (!lodash.isNil(event.target.closest('.actions-more-info'))) {
```
merge it with the condition below

mavdryk commented on src/igz_controls/less/table.less:
```diff
@@ -76,6 +76,10 @@
         }
+
+        &.invocation {
+            overflow: visible;
```
because of this line, the ellipsis of the cell won't work

mavdryk commented on src/nuclio/functions/function-collapsing-row/function-collapsing-row.component.js:
```diff
@@ -208,6 +210,9 @@
                 });
 
                 NuclioHeaderService.updateMainHeader('common:PROJECTS', ctrl.title, $state.current.name);
+            } else if (!lodash.isNil(event.target.closest('.actions-more-info'))) {
+                event.stopPropagation();
+                return;
```
that's not what I meant. Extend `if` instead of adding `else if`

mavdryk commented on src/nuclio/functions/function-collapsing-row/function-collapsing-row.tpl.html:
```diff
@@ -68,7 +68,7 @@
                 {{$ctrl.runtimes[$ctrl.function.spec.runtime]}}
             </div>
-            <div class="igz-col-12-5 common-table-cell"
+            <div class="igz-col-12-5 common-table-cell invocation"
```
unused class `invocation`

iurii-vatazhyshyn committed:
Fix [Nuclio] Actions remain disabled after deploy cancel/fail (#1140)

eran-nussbaum committed:
Impl [Build] HTML: Collapse consecutive spaces into 1, not 0 (#1141)

eran-nussbaum committed:
Create a new dist version 0.28.10 (#1142)

Collecting:
- Impl [Nuclio] Allow overriding when a newer version exists (#1138)
- Fix [Nuclio] Create function: pagination not working on filter (#1139)
- Fix [Nuclio] Actions remain disabled after deploy cancel/fail (#1140)
- Impl [Build] HTML: Collapse consecutive spaces into 1, not 0 (#1141)

eran-nussbaum committed:
Impl [General] Inherit font style of <blockquote> from its parent(#1143)

eran-nussbaum committed:
Create a new dist version 0.28.11 (#1144)

Collecting:
- Impl [General] Inherit font style of <blockquote> from its parent
  (#1143)

eran-nussbaum committed:
Fix [igzDefaultDropdown] Potential whitespace bug (#1147)

eran-nussbaum committed:
Impl [Nuclio] Function: add unique classname to stale version msg (#1148)

`function-deploy-stale`

iurii-vatazhyshyn committed:
Impl [Nuclio] Configuration: remove "FS group" field on K8s (#1146)

iurii-vatazhyshyn committed:
Impl [API gateways] Get authentication type options from backend (#1145)

eran-nussbaum committed:
Create a new dist version 0.28.12 (#1149)

Collecting:
- Fix [igzDefaultDropdown] Potential whitespace bug (#1147)
- Impl [Nuclio] Function: add unique classname to stale version msg
  (#1148)
- Impl [Nuclio] Configuration: remove "FS group" field on K8s (#1146)
- Impl [API gateways] Get authentication type options from backend
  (#1145)

eran-nussbaum committed:
Fix [Nuclio] Build configuration: image prefix gone after deploy (#1150)

eran-nussbaum committed:
Create a new dist version 0.28.13 (#1151)

Collecting:
- Fix [Nuclio] Build configuration: image prefix gone after deploy
  (#1150)

Commit:
[Nuclio] Invocation URL: N/A on error or not exposed, +info1
